RHOCore.js

```diff
@@ -160,3 +160,17 @@ function toRholang(par /*: IPar */) /*: string */ {
 
   return recur(par);
 }
+
+
+exports.rhol = rhol;
+function rhol(template /*: string[] */, ...subs /*: Json[] */) {
+  const encoded = subs.map(it => toRholang(fromJSData(it)));
+
+  const out = [];
```

Comment:
should this be `const` if you are pushing things onto it?

Reply:
Though the object it refers to is mutable. the binding between const and the object it refers to is never re-assigned.

There are "don't use const" rants about this, and it would be nice if JavaScript had immutable datastructures in its stdlib, but I'm content with this style of code.